src/wallet/components/Wallet.tsx

```diff
@@ -60,6 +72,10 @@ function WalletContent({
   useOutsideClick(walletContainerRef, handleClose);
 
   const { connect, dropdown, advanced } = useMemo(() => {
+    // default children implementation
+    if (!children) {
+      return defaultWalletChildren;
```

Comment:
note: Because the default children will have referential equality across renders (since they're defined outside the component), they won't rerender when the parent rerenders. This is almost certainly fine in this case, but just wanted to call it out since it can be unexpected.